tut/installing-appliance.md

```diff
@@ -35,15 +35,29 @@ Run the following commands to ensure that all installed packages on your system
     sudo apt update
     sudo apt upgrade
 
-### 2. Attach your machine to the UA subscription
+### 2. Install dependencies
+
+LXD >= 5.0 is required for the Anbox Cloud Appliance. Check which version you have currently installed:
+
+    lxd --version
+
+If LXD is not installed, run
+
+    snap install --channel=5.0/stable lxd
+
+If LXD is already installed but the version is older than 5.0, run
+
+    snap refresh --channel=5.0/stable lxd
+
+### 3. Attach your machine to the UA subscription
 
 The Anbox Cloud Appliance requires a valid Ubuntu Advantage for Applications subscription.
 
 Before installing the appliance, you must attach the machine on which you want to run the Anbox Cloud Appliance to your Ubuntu Advantage for Applications subscription. To do so, run the following command, replacing *<UA_token>* with your Ubuntu Advantage for Applications token:
 
     sudo ua attach <UA_token>
 
-### 3. Install the snap
+### 4. Install the snap
```

Comment:
There's a "4. Install additional tools" underneath that must be "5." now.

Reply:
Fixed